test/tests/resilience/pbs_hook_alarm_large_multinode_job.py

```diff
@@ -59,7 +59,7 @@ def setUp(self):
         # Restart mom explicitly due to PP-993
         self.mom.restart()
 
-    @timeout(400)
+    @timeout(1800)
```

Comment:
This is a pretty serious increase from 6.5m to 30m.  Do you really think these tests will take that long?